### DISCUSSION TITLE
Bug or Account Security?

### DISCUSSION THREAD

**Discussion by kjxbyz** [community/community#148112] [Mobile] [answered]
### Select Topic Area

Question

### Body

The last comment was not created by me, but why is it shown as my comment? Is this a bug or an issue with account security?

![IMG_2661](https://github.com/user-attachments/assets/f3f0039c-b7d9-4f7e-9374-18f99f402648)

**Comment by iskae** [comment, marked as answer]
Hi @kjxbyz 👋 

Thank you so much for reaching out. I believe I understood what is going on here. We are showing `You commented` in there whenever the users last interaction was a comment to the issue/PR/discussion. This does not mean though the latest comment was from you, it just means that your recent activity on this item was a comment 😊 

I hope this brings some clarity around this issue 🙇 Please let us know if we can improve things for you in the future 😊 

Cheers!
